### PR TITLE
Created two other suit colors to help differentiate Orion Armor wearers

### PR DESCRIPTION
**Pull request by lngredients**
There is now red, blue, and green orion armors

### CODE REVIEW

**Comment by lngredients** [comment]
Hopefully they don't look dumb

File: code/modules/halo/misc/Orion Project.dm [hunk 1]
```diff
@@ -19,9 +19,11 @@
 	desc = "A prototype thick, layered grey undersuit lined with power cables and flexible nano composite plates."
 	species_restricted = list("Orion")
 
-/obj/item/clothing/suit/space/void/swat/theta
-	name = "Orion Armor"
+/obj/item/clothing/suit/space/void/swat/thetar
```
Comment (caelaislinn) [inline code/modules/halo/misc/Orion Project.dm:22]: Changing these file paths breaks the geminus city map... you'll need to either update the file paths in the map or add the coloured variants as a different path (preferably a subclass)

**Review by caelaislinn — CHANGES_REQUESTED** [review]

File: code/modules/halo/misc/Orion Project.dm (hunk 1 above)
Comment (lngredients) [reply code/modules/halo/misc/Orion Project.dm:22]: I don't know why there is theta armor in gemcity in the first place, I'll look into it.

Comment (caelaislinn) [reply code/modules/halo/misc/Orion Project.dm:22]: Weren't you the one who added it?

Comment (lngredients) [reply code/modules/halo/misc/Orion Project.dm:22]: No, I didn't add the theta armor in gemcity, it should have been removed in fact.

**Pull request closed without merging**